kornia/feature/integrated.py

```diff
@@ -114,12 +118,16 @@ class LocalFeature(nn.Module):
     Args:
         detector: the detection module.
         descriptor: the descriptor module.
+        scaling_coef: multiplier for change default detector scale (e.g. it is too small for KeyNet by default)
     """
 
-    def __init__(self, detector: nn.Module, descriptor: LAFDescriptor) -> None:
+    def __init__(self, detector: nn.Module, descriptor: LAFDescriptor, scaling_coef: float = 1.0) -> None:
         super().__init__()
         self.detector = detector
         self.descriptor = descriptor
+        if scaling_coef <= 0:
+            raise ValueError(f"Scaling coef should be >= 0, got {scaling_coef}")
```

Comment:
`KORNIA_CHECK` ?

Reply:
KORNIA_CHECK is for tensors, this is just plain check